probe/docker/reporter.go

```diff
@@ -56,8 +58,10 @@ var (
 			Label: "Image",
 			Type:  report.PropertyListType,
 			FixedRows: map[string]string{
-				ImageID:          "ID",
-				ImageName:        "Name",
+				// Prepend spaces as a hack to keep at the top when sorted.
+				ImageID:          " ID",
+				ImageName:        " Name",
```

Comment:
Not sure if per the OCI spec, it might be Image repo instead of name? Didn't have a chance to have a look.

Reply:
I'd go for name, even if we can have $repo/$name sometimes here. There are also special cases for well known images (`busybox`, `golang`, ...).